Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM ubuntu:18.04
+RUN apt-get update -y
+RUN apt-get install -y python3-pip python3-setuptools libsystemd-dev
```

Comment:
Merge the RUN commands into one chained RUN instruction.
You can find the reasoning for that in the following KB:
https://stackoverflow.com/questions/39223249/multiple-run-vs-single-chained-run-in-dockerfile-what-is-better